talentsearch/resources/js/api/poolCandidateFilterOperations.graphql

```diff
@@ -57,7 +57,7 @@ query getPoolCandidateFilters {
   }
 }
 
-query getPoolCandidateFilterData($id: ID!) {
+query getSearchFormData {
```

Comment:
This query is failing for me without authentication.  Maybe some guards need to be removed to allow this to work?

Reply:
Okay, I removed the gaurds and updated the policies so that the lookup tables (classifications, cmo assets, etc.) are public.